src/PostRector/Rector/NameImportingPostRector.php

```diff
@@ -20,6 +20,11 @@
 
 final class NameImportingPostRector extends AbstractPostRector
 {
+    /**
+     * @var array<Use_|GroupUse>
+     */
+    private array $currentUses;
```

Comment:
better to have default empty array to avoid invalid usage as not initiliazed by `__construct()`

```suggestion
    private array $currentUses = [];
```